.ci/deps.pip.sh

```diff
@@ -4,17 +4,17 @@ set -x
 TERM=dumb
 
 # Choose the python versions to install deps for
-case $CIRCLE_NODE_INDEX in
- 0) dep_versions=( "3.4.3" "3.5.1" ) ;;
- 1) dep_versions=( "3.4.3" ) ;;
- -1) dep_versions=( ) ;;  # set by .travis.yml
- *) dep_versions=( "3.5.1" ) ;;
+case $CIRCLE_JOB in
+  "python-3.5") dep_versions=( "3.4.3" "3.5.1" );;
+  "python-3.4") dep_versions=( "3.4.3" );;
+  *)  dep_versions=( "3.5.1" );;
 esac
 
+pyenv install -ks 2.7.10
```

Comment:
why installing 2.7.10?

Reply:
in circleci 1.0 python 2.7.10 is already installed
And we have a command 
   pyenv local $dep_version 2.7.10
which doesnot work without installing this version
And i thought there might be some portion of code which is dependent on it

Should I just find the default version of python 2 installed and change this command to
   pyenv local $dep_version $installed_version